package.json

```diff
@@ -2,15 +2,20 @@
   "repository": {},
   "license": "MIT",
   "scripts": {
+    "precommit": "lint-staged",
     "clean-hot-updates": "find . -name '*.hot-update*' -delete",
     "compile-production": "webpack -p --config='webpack.dll.js' --hide-modules && webpack -p",
     "compile-dev": "webpack --config='webpack.dll.js' --hide-modules && webpack --hide-modules",
     "watch": "webpack --config='webpack.dll.js' --hide-modules && NODE_ENV=dev node webpack.devserver.js",
+    "precommit": "lint-staged",
```

Comment:
hey @dplain90, overall the PR looks great. somehow this npm script addition was added twice, so gonna do a follow-up commit removing one them